main.cpp

```diff
@@ -1,17 +1,64 @@
 #include "main.h"
 #include "rc.h"
 #include <filesystem>
+#include <gdiplus.h>
 
-void main_window::on_paint(HDC hdc) 
-{
+void main_window::on_paint(HDC hdc) {
+	RECT rect;  // Lef, top, right, bottom
+	GetClientRect(*this, &rect);
+	Gdiplus::RectF layoutRect( rect.left, rect.top, rect.right, rect.bottom);  // X, Y, Width, Height
+	Gdiplus::Graphics graphics(hdc);
+
+	// Image
+	graphics.DrawImage(image, layoutRect);
+
+	if (image) {
```

Comment:
Zašto `DrawImage` nije pod ovim `if`-om?
Zašto ne samo na početku funkcije
`if (!image) return;`

Reply:
Ne sjećam se što sam eksperimentirao s tim if-om.